tool/src/cpp2/ty.rs

```diff
@@ -326,7 +309,11 @@ inline {type_name} {type_name}::FromFFI({ctype} c_struct) {{
         for param in method.params.iter() {
             let decls = self.gen_ty_decl(&param.ty, param.name.as_str());
             param_decls.push(decls);
-            let conversions = self.gen_cpp_to_c_for_type(&param.ty, param.name.as_str());
+            let conversions = self.gen_cpp_to_c_for_type(
+                &param.ty,
+                "<UNUSED>".into(),
```

Comment:
nit: might be worth making this an Option?

Reply:
Done by making this version of the function not require both parameters